CsvParser/csvParser.go

```diff
@@ -81,20 +80,19 @@ func CsvParser(filePath string, numIncidents ...int) []incident.DefaultIncident
 		avgBadDString := strings.Split(rec[7], " ")
 		avgBadDS, _ := strconv.ParseFloat(avgBadDString[1], 64)
 
+		// Make an instance of a DefaultIncident that is compatible with the Incident interface
 		defaultIncident := new(incident.DefaultIncident)
-
-		defaultIncident.Init(goodTimeStart, goodTimeEnd, timeStart, timeEnd, avgGoodDS,
-			avgBadDS, severity, testsAffected)
+		defaultIncident.MakeIncidentData(goodTimeStart, goodTimeEnd, timeStart, timeEnd, avgGoodDS, avgBadDS, severity, testsAffected)
```

Comment:
looks like this was also in the previous code, but are timeStart and timeEnd the start time and end time of the bad period?

Reply:
definitely not my call, but wondering if the variables should be called badTimeStart and badTimeEnd?

Reply:
also, nitpicky but in MakeIncidentData average good download speeds and average bad download speeds are called "avgDSGood" and "avgDSBad", respectively... might be helpful for readability to keep them consistent.

Reply:
Agreed about keeping variable names consistent here!

Reply:
Yeah I believe Pascal wrote that part of the CSV parser. It is to indicate the start and end of the incident (which is implicitly the bad period in this file). Changed the variable names to be more consistent, thanks!